app/components/common/ApiJsonFactory.js

```diff
@@ -5,14 +5,18 @@
 var commonFactories = angular.module('openevent');
 
 commonFactories.factory('ApiJsonFactory', ['$q', '$http', function ($q, $http) {
-var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl)
-                + 'event/' + config.eventId + '/';
+var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl) +
+    'event/' + config.eventId;
 
         return {
             getJson: function ($apiEndpoint) {
-                var deferred = $q.defer(),
-                    httpPromise = $http.get(baseUrl + $apiEndpoint);
-
+                if (($apiEndpoint === 'event') && (!config.use_testApi)) {
```

Comment:
@championswimmer could make the use_testApi in camel case? then we can get rid of this error.